foodsaving/users/api.py

```diff
@@ -42,6 +42,9 @@ def get_permissions(self):
             self.permission_classes = (AllowAny,)
         elif self.action in ('list', 'retrieve'):
             self.permission_classes = (IsAuthenticated,)
+        else:
+            # Issue 232 default permission classes should always be returned
+            self.permission_classes += self.__class__.permission_classes
```

Comment:
right now in case of /api/users/verify_mail/ self.permission_classes return 
`(<class 'foodsaving.users.permissions.IsNotVerified'>, <class 'rest_framework.permissions.IsAuthenticated'>)` 
this tuple is not returning default permission class which is defined at Class level.
and with the above code ... self.permission_classes return 

`(<class 'foodsaving.users.permissions.IsNotVerified'>, <class 'rest_framework.permissions.IsAuthenticated'>, <class 'foodsaving.users.permissions.IsSameUser'>)`

here IsSameUser is permission classes which is defined at class level and I am assuming the classes which is defined at class level is default permission and should always be returned.